hack/build.sh

```diff
@@ -58,20 +58,19 @@ function digest() {
 	ret=$(docker inspect --format='{{index .RepoDigests 0}}' "${IMAGE}")
 }
 
-docker build -t "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}" .
-docker push "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}"
+docker buildx build --push --platform "linux/amd64,linux/ppc64le"  -t "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}" .
 digest "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}" OPERATOR_DIGEST
 
 digest "${REGISTRY}/${NAMESPACE}/quay:${TAG}" QUAY_DIGEST
 digest "${REGISTRY}/${NAMESPACE}/clair:nightly" CLAIR_DIGEST
 digest "${REGISTRY}/${NAMESPACE}/quay-builder:${TAG}" BUILDER_DIGEST
-digest "${REGISTRY}/${NAMESPACE}/quay-builder-qemu:main" BUILDER_QEMU_DIGEST
+digest "${REGISTRY}/${NAMESPACE}/quay-builder-qemu:3.9.0" BUILDER_QEMU_DIGEST
 # shellcheck disable=SC2034
-POSTGRES_DIGEST='centos/postgresql-13-centos7@sha256:71b24684d64da46f960682cc4216222a7e4ed8b1a31dd5a865b3e71afdea20d2'
+POSTGRES_DIGEST='quay.io/sclorg/postgresql-13-c9s@sha256:593910f2d4b895f4924261a3b8b2aa6457892100a01a0c0ad661cd378d810d65'
```

Comment:
@jonathankingfc will upgrades work with this image? won't it break downstream?

Reply:
the downstream images are defined in the midstream cpaas repo, so this won't affect the downstream builds